research-hub-web/src/app/components/shared/body-media/body-media.component.scss

```diff
@@ -209,12 +209,15 @@
  */
 .table-container {
```

Comment:
might have to look at changing the table layout to overflow/scrolling because it's looking pretty squashed on mobile view